keyboards/akko/5108/keymaps/via/rules.mk

```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+ENCODER_MAP_ENABLE = no
```

Comment:
```suggestion

```
Since this board doesn't look like it's using an encoder, this flag shouldn't be needed